### ISSUE TITLE
Issue with hex-input slot

### ISSUE THREAD

**Issue by Reidond** [web-padawan/vanilla-colorful#65]
Hello,

This html `<hex-input><input class="form-control form-unit__input mb-1" part="input" spellcheck="false"></hex-input>`

Produces this:
![image](https://user-images.githubusercontent.com/20406775/133891128-9933279c-d393-42e2-81eb-f4379d49d325.png)

DOM nodes:
![image](https://user-images.githubusercontent.com/20406775/133891145-dffe6fc5-3fb0-46bd-8676-20de380c12f3.png)

**Comment by web-padawan** [comment]
Hi, please use `slot` attribute to customize the input passed to light DOM:

```html
<hex-input><input class="form-control form-unit__input mb-1" slot="input" spellcheck="false"></hex-input>
```

**Comment by Reidond** [comment]
is it possible that, if i have two hex-inputs on page, component add 2 slots?

**Comment by Reidond** [comment]
because i have one component that works great, and one that generates this strange DOM nodes (2 pic in issue), but they are absolutely same

**Comment by web-padawan** [comment]
Should be fixed by #93 released in 0.7.2.
